modules/persistent-storage-csi-group-snapshots-overview.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-group-snapshots.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-group-snapshots-overview_{context}"]
+= Overview of CSI volume group snapshots
```

Comment:
this goes TP in 4.18 please add a TP warning as well as the usual procedure to enable tech preview no upgrade
thanks